components/omega/src/infra/logging.cpp

```diff
@@ -0,0 +1,13 @@
+// Omega logging implementation file.
+
+#include "logging.h"
+
```

Comment:
We want wrap inside namespace OMEGA here

Reply:
OK